tests/lib/__init__.py

```diff
@@ -380,7 +369,7 @@ def pip(self, *args, **kwargs):
         if (pyversion_tuple < (2, 7, 9) and
                 args and args[0] in ('search', 'install', 'download')):
             kwargs['expect_stderr'] = True
-        if kwargs.pop('use_module', False):
+        if kwargs.pop('use_module', True):
```

Comment:
Any specific reasons for this?

Reply:
I had noticed the removal of the launchers but I was wondering why that was done (and thus this change).

I'm not sure I understand why they're not relocatable, perhaps @pfmoore has a better idea about that than me.